docs/src/reference/1_algorithms.md

```diff
@@ -0,0 +1,12 @@
+# Algorithms 
```

Comment:
Algorithm 

Not Algorithms. Boscia is one algorithm with many options and settings.